src/applications/personalization/components/NameTag.jsx

```diff
@@ -141,8 +142,9 @@ const NameTag = ({
             {showUpdatedNameTag &&
               totalDisabilityRating && (
                 <>
-                  <dt className="sr-only">total disability rating</dt>
+                  <dt className="sr-only">your disability rating</dt>
```

Comment:
updated this to match the new `"your disability rating:"` intro text added on line 147